CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ Removed
 * Haskell related tasks in favor of using an external dedicated role.
 
 # CHANGELOG
+
+## 1.7.4
+- Remove Fail2Ban role due to the high disk space usage of the resulting log
```

Comment:
Also, could you add a link to the role, please? We should also mention that we are getting rid of this dependency because it was not updated in several years